### PR TITLE
Update to use pipenv

### PR DESCRIPTION
**Pull request by msoucy**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+name = "pypi"
+
+[dev-packages]
+
+[packages]
+yamllint = "*"
+"Jinja2" = "*"
+
+[requires]
+python_version = "2.7"
```
Comment (jflory7) [inline Pipfile]: In 55594a19277c0a2f231dc1134377313fe2ea3572, we added dual-support for Python 2.7 and 3.6. Our CI also tests for both versions. Could you also make sure Python 3.6 is included for testing?

File: Pipfile.lock [hunk 2]
```diff
@@ -0,0 +1,68 @@
+{
+    "_meta": {
+        "hash": {
+            "sha256": "92c271cf58b7b905ee81b93ca29f1fdf58b0b620f09c22a64d17686c6078fb21"
+        },
+        "pipfile-spec": 6,
+        "requires": {
+            "python_version": "2.7"
```
Comment (jflory7) [inline Pipfile.lock]: Same note on Python 3, since we are looking towards the Python 2 EoL soon anyways.

**Comment by msoucy** [comment]
Closing, since Travis doesn't support pipenv at the moment, and I don't have the energy to try to shim it in... Sorry for the tease

**Comment by jflory7** [comment]
@msoucy No worries, thanks anyways. :+1:  Want to file an issue for it to keep it on the radar?

**Comment by msoucy** [comment]
http://xkcd.com/1987 is very relevant...

**Pull request closed without merging**